13/main/index.html

```diff
@@ -0,0 +1,1480 @@
+<!DOCTYPE HTML>
+<html class="no-js" xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US" >
+ <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# felicity-chatbot: http://ogp.me/ns/fb/felicity-chatbot#">
+  <meta property="fb:app_id" content="143418345817318" />
+  <meta property="og:type"   content="felicity-chatbot:chatbot_question" />
+  <meta property="og:url"    content="http://felicity.iiit.ac.in/13/main/index.php" />
+  <meta property="og:title"  content="Felicity Chatbot Question" />
+  <meta property="og:image"  content="http://felicity.iiit.ac.in/13/main/images/felicity_Logo.png" />
+  <script type="text/javascript" src="js/jquery.min.js"></script>
+  <script type="text/javascript" src="js/jquery-ui.min.js"></script>
+  <style type='text/css'>
+#SPONSORS li u h3 {
+font-family:"'Trebuchet MS', arial, sans-serif;"
+}
+#SPONSORS img {
+padding:5px;
+}
+</style>
+  <script type="text/javascript">
+
+
+  showQs = function (){
+    $.ajax({type:'GET',url:'/web2py/ChatBot_V4/default/count_question',success:function(temp){
+                    if(temp) { document.getElementById("noOfQs").innerHTML = temp;}
+                    else alert("booo"+temp);}});
+
+  }
+  function postQuestion()
+  {
+
+  question = document.getElementById("question_box").value;
+FB.api(
+  '/me/felicity-chatbot:ask',
+  'post',
+  {
+    ch_question_text: question,
+    chatbot_question: "http://felicity.iiit.ac.in/13/main/index.php"
+  },
+  function(response) {
+     alert(response.error);
+  /*if(!response || response.error){
+     //alert("Please 'Like' before sharing.\nIf you have just 'Liked' it, please refresh the page.");
+  }
+  else{
+      $(".uibutton").hide(300);
+  }
+      //Nothing to do
+   */
+  }
+);
+  }
+  </script>
+
+
+
+<title> Felicity 2k13  </title>
+<link rel="stylesheet" type="text/css" href="style/style.css" media="all" />
+<link rel="stylesheet" type="text/css" href="style/fb_style.css" media="all" />
+<script type="text/javascript" src="js/modernizr.js"></script>
+<script type="text/javascript" src="js/get_answer_GHJGDJHGJDHG.js"></script>
```

Comment:
I think the get_answer_<string> is a randomly generated string.

But I suppose that this would do for now.